release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -75,5 +109,6 @@ def consume_fn(ds):
 
 
 if __name__ == "__main__":
+    ray.init()
```

Comment:
OOC why did we need to add this here?

Reply:
So that Ray initialization time isn't included in the benchmark time

Reply:
Ah makes sense, thanks!